docs/extend/install-configure-zos-extensions.md

```diff
@@ -1,40 +1,72 @@
-# Install, upgrade and configure Zowe server component
+# Install, upgrade, and configure Zowe server component
 
-Learn how to install, upgrade and configure the Zowe server components or extensions manually or by using scripts `zowe-install-component.sh`, `zowe-upgrade-component.sh` and `zowe-configure-component.sh`. 
+Learn how to install, upgrade, and configure the Zowe server components or extensions manually or by using the following scripts: 
+* `zowe-install-component.sh`
+* `zowe-upgrade-component.sh` 
+* `zowe-configure-component.sh`. 
 
 ## Install with `zowe-install-component.sh` (Technical Preview)
 
 <Badge text="Technical Preview"/>
 
-_Note: This section is for technical preview and we are happy to hear any feedback. Content in this section may be changed or improved in the future._
+**Notes:** 
+* This section is for technical preview. As such, we welcome  feedback. Content in this section may be changed or improved in the future.
+
+* This feature is added with Zowe v1.19.0 release.
+
+From Zowe v1.19.0, Zowe ships a `bin/zowe-install-component.sh` tool to help you install any Zowe server component (extension). Zowe core components are also installed with this utility. In order to be compatible with the utility, we recommend components follow [Zowe server component package format standard](packaging-zos-extensions.md#zowe-server-component-package-format).
+
+Execute the utility from z/OS USS, Use the following command line parameters:
```

Comment:
It should be: 
`Execute the utility from z/OS USS. Use the following command line parameters:`